common/djangoapps/student/management/commands/transfer_students.py

```diff
@@ -52,7 +52,6 @@ class Command(TrackedCommand):
                     help="If True, try to transfer certificate items to the new course.")
     )
 
-    @transaction.commit_manually
```

Comment:
BTW, @symbolist - have you seen @nedbat 's following commit?:

https://github.com/edx/edx-platform/commit/85c57f4fb54c7352f1e3b796be46ad01b0fce17d

It changes several `commit_on_success` to `atomic`.